Formula/convco-cli.rb

```diff
@@ -0,0 +1,32 @@
+class ConvcoCli < Formula
+  desc "Conventional commits, changelog, versioning, validation"
+  homepage "https://convco.github.io"
+  url "https://github.com/convco/convco/archive/refs/tags/v0.4.0.tar.gz"
+  sha256 "88c92b175163c8847da7dd201d32106c51ff85e2c992c2a3ff67e29ecbe57abb"
+  license "MIT"
+  head "https://github.com/convco/convco.git", branch: "master"
+
+  livecheck do
+    url "https://github.com/convco/convco/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
```

Comment:
Doesn't seem like this is needed. Have you tried dropping it and checking the output from `brew livecheck`?

Reply:
As Carlo suggested, the `livecheck` block isn't necessary here -- `brew livecheck` works just fine without it.

Reply:
```
$ brew livecheck convco
Error: convco: Unable to get versions
```